src/test/java/org/apache/commons/lang3/StringUtilsEmptyBlankTest.java

```diff
@@ -99,6 +110,7 @@ public void testIsNotBlank() {
         assertFalse(StringUtils.isNotBlank(null));
         assertFalse(StringUtils.isNotBlank(""));
         assertFalse(StringUtils.isNotBlank(StringUtilsTest.WHITESPACE));
+        assertTrue(StringUtils.isNotBlank("a"));
```

Comment:
I might be missing something, but this appears redundant with the next line. Did this add any coverage?

Reply:
This is a special case that fails if you accidentally use `index > 0` instead of `index >= 0` in the expression of the down going loop. The next line will not fail in this case.

Reply:
Might be worth adding a comment in there for that.

Reply:
Done in second commit.